environment_setup/install_requirements.sh

```diff
@@ -24,8 +24,7 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THE SOFTWARE CODE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-
-python --version
-pip install azure-cli==2.0.46
-pip install --upgrade azureml-sdk[cli]
-pip install -r requirements.txt
\ No newline at end of file
+set -eux
+pip install conda-merge==0.1.5
+conda-merge diabetes_regression/training/training_dependencies.yml diabetes_regression/scoring/scoring_dependencies.yml environment_setup/ci_environment.yml > /tmp/conda_merged.yml
+conda env create -f /tmp/conda_merged.yml
```

Comment:
please activate the new conda env after creation

Reply:
Done https://github.com/microsoft/MLOpsPython/pull/158/commits/921b65f9e1c7a19f8aa7271db7a2a9b910ebb7cf